### ISSUE TITLE
DataLoader

### ISSUE THREAD

**Issue by MSiam** [ElementAI/wise_ils#1]
Hello

   Would it be possible to describe in your readme where did you get 
pascal_val2012.json
lcfcn_points/Pascal2012.pkl

Thanks

**Comment by IssamLaradji** [comment]
Hi @MSiam ,

1. I realized `pascal_val2012.json` is not needed, so I removed it. Anyhow, it can be found here: 
http://cocodataset.org/#external,
The direct link for downloading it is:
https://storage.googleapis.com/coco-dataset/external/PASCAL_VOC.zip

2. `lcfcn_points` is not needed as well for this project, I removed it.

What is needed to get the validation labels is the path for the pascal folder. I added the possibility to add that path as an input argument for the dataset. The path to where Pascal stuff is saved is usually something like `VOCdevkit/VOC2012/`.

**Comment by MSiam** [comment]
Thanks. is it used in the training phase?

**Comment by IssamLaradji** [comment]
No, I use `prm_points` instead. I am uploading the training soon! `lcfcn_points` are points  generated by the lcfcn method `https://github.com/ElementAI/LCFCN` which requires point-level supervision. It was used for comparison.

**Comment by IssamLaradji** [comment]
training loop has been added! you can train mrcnn using `python train.py`

**Comment by MSiam** [comment]
Thanks
